com.transmissionbt.Transmission.json

```diff
@@ -0,0 +1,66 @@
+{
+    "app-id": "com.transmissionbt.Transmission",
+    "branch": "master",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.24",
+    "sdk": "org.gnome.Sdk",
+    "command": "transmission-gtk",
+    "rename-desktop-file": "transmission-gtk.desktop",
+    "rename-icon": "transmission",
+    "copy-icon": true,
+    "finish-args": [
+        "--share=ipc",
+        "--share=network",
+        "--socket=x11",
+        "--socket=wayland",
+        "--socket=session-bus",
```

Comment:
I'm sure it's vanishingly uncommon. I guess we could patch [https://github.com/transmission/transmission/blob/ed754f96051b9f761ad0891b92ae5bee05b7538e/gtk/main.c#L675-L679](this block) to unconditionally use `com.transmissionbt.Transmission`, or use that unless `--config-dir` is passed.

Related: upstream's bus name has a lowercase `t` – `com.transmissionbt.transmission_%lu_%lu` – but I named this Flatpak with the uppercase T which seems more conventional. Any preference here? Interestingly the Qt frontend does not have this `--config-dir` behaviour and claims the name `com.transmissionbt.Transmission` with a capital T. It uses the same config so running both at once seems questionable at best.